components/scream/src/dynamics/homme/atmosphere_dynamics.cpp

```diff
@@ -315,6 +315,25 @@ void HommeDynamics::initialize_impl ()
 
   // Complete homme model initialization
   prim_init_model_f90 ();
+
```

Comment:
What do you think about putting these inside a `add_property_checks` method, as you've done for all the above operations? That might make for an easier setting in which someone can digest the assumptions.

One could argue that we could provide an `add_property_checks` method as part of the `AtmosphereProcess` interface to encourage people to think about it in this way, but I don't know if that's necessary.

Reply:
Yeah, it's definitely not needed, with the `add_property_check` method already there. I like this reorganization, Luca.